frontend/src/components/pages/CampOverview/EditCamperModal/index.tsx

```diff
@@ -131,7 +130,6 @@ const EditCamperModal = ({
         <ModalHeader>
           Edit {camper.firstName} {camper.lastName}&apos;s Information
         </ModalHeader>
-        <ModalCloseButton />
```

Comment:
just wanted to confirm the reason for removing this button does it just not match the designs?

Reply:
Yeah, the modals don't have the close button (consistency).